File: app/src/main/java/com/scottmangiapane/cardclicks/PausedActivity.java
package com.scottmangiapane.cardclicks;

import android.os.Bundle;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;

public class PausedActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_paused);
        findViewById(R.id.button_resume_game).setOnClickListener(v -> finish());
    }
}

File: app/src/main/java/com/scottmangiapane/cardclicks/PausedActivity.kt
package com.scottmangiapane.cardclicks

import android.os.Bundle
import android.view.View
import androidx.activity.OnBackPressedCallback
import androidx.appcompat.app.AppCompatActivity

class PausedActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_paused)
        findViewById<View>(R.id.button_resume_game).setOnClickListener { finish() }
        onBackPressedDispatcher.addCallback(this, object : OnBackPressedCallback(true) {
            override fun handleOnBackPressed() {
                moveTaskToBack(true);
            }
        })
    }
}
